### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by michael-suh**
js code journal 1 SS
js code journal 1 SR

### CODE REVIEW

**Comment by michael-suh** [comment]
js code journal 1 mobile SR

File: index.html [hunk 1]
```diff
@@ -10,15 +10,51 @@
...
+              <label class="full-width headings" for="title">Title</label>
+              <input class="full-width text-input title" type="text" required="required">
+            </div>
+            <div class="row">
+              <label class="full-width headings" for="photourl">Photo URL</label>
+              <input class="full-width text-input photourl" type="url" required="required">
+            </div>
+          </div>
+
+          <div class="column-full row">
```
Comment (codingcodymiller) [inline index.html]: What is the purpose of the `row` class here? I don't see any `column` class elements as children of this. Also, no one element should ever be both a `column` and a `row`. If you need a `row` in a `column`, you should make it a child element of the `column`.

File: index.html [hunk 2]
```diff
@@ -10,15 +10,51 @@
...
 </head>
 <body>
-  <header>
-  </header>
-  <main>
-  </main>
+  <div class="center">
+
+    <header class="headers">
+      <nav class="container row headers full-width header-position">
```
Comment (codingcodymiller) [inline index.html]: No one element should be both a `container` and a `row`. You should make the `row` a child of the `container`

File: index.html [hunk 3]
```diff
@@ -10,15 +10,51 @@
...
+        <h1 class="headers main-header headings">Code Journal</h1>
+      </nav>
+    </header>
+
+    <main class="container full-width">
```
Comment (codingcodymiller) [inline index.html]: What is this `full-width` class you are using? I don't see it in your styling anywhere. Do you mean to be using the `column-full` class we prepared for you in `layout.css`?

File: js/main.js [hunk 4]
```diff
@@ -1,2 +1,23 @@
...
+}
+
+$photoURL.addEventListener('input', updateImg);
+
+var $form = document.querySelector('form');
+
+function updateEntry(event) {
+  event.preventDefault();
+  var entriesObj = {};
+  data.nextEntryId++;
```
Comment (codingcodymiller) [inline js/main.js]: This object doesn't include any data from the form elements, it needs a property for each form control element in the form.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**